orchestrator/consumers/chatbot/smart_memory.py

```
@@ -480,26 +443,34 @@ async def store_conversation(
-                    results = await self.unified_service.store_two_tier(
+                    fact_results = await self.unified_service.store_two_tier(
                         workspace_id=workspace_id,
-                        messages=l3_messages,
+                        messages=[{"role": "user", "content": fact["fact"][:max_chars]}],
                         agent_id=agent_id,
                         tier=tier,
-                        metadata=base_metadata,
+                        metadata=fact_meta,
                     )
+                    results.extend(fact_results)
                 except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)